lambdas/services/bulk_upload_service.py

```diff
@@ -29,8 +34,9 @@ def __init__(
         self.staging_bucket_name = os.environ["STAGING_STORE_BUCKET_NAME"]
         self.lg_bucket_name = os.environ["LLOYD_GEORGE_BUCKET_NAME"]
         self.lg_dynamo_table = os.environ["LLOYD_GEORGE_DYNAMODB_NAME"]
+        self.bulk_upload_report_dynamo_table = os.environ["BULK_UPLOAD_DYNAMODB_NAME"]
         self.invalid_queue_url = os.environ["INVALID_SQS_QUEUE_URL"]
-
+        self.metadata_queue_url = os.environ["METADATA_SQS_QUEUE_URL"]
```

Comment:
Are we catching KeyError somewhere (that is not in the general Exception in the lambda)?

Reply:
No, because this lambda isn't a web api, so I don't put anything to catch missing env var here.
If any env var is missing from here, the lambda would fail immediately during the `bulk_upload_service = BulkUploadService()`